lib/minesweeper_2pl/app.rb

```diff
@@ -1,7 +1,41 @@
-module Minesweeper_2pl
+module Minesweeper
   class App
-    def call
-      "Hello World"
+    BOMB_PERCENT = 0.75
+    MAX_ROW_NUM = 20
+
+    attr_accessor :game, :cli
+
+    def initialize
+      cli = CLI.new
```

Comment:
CLI is an OK name, but will this game always run on the command line? Does it make sense to have a CLI when you use this gem in a Rails application?